### DISCUSSION TITLE
Live TV

### DISCUSSION THREAD

**Discussion by NetSven** [jellyflix-app/jellyflix#100] [Ideas]
Can you put in LIVE TV ? Thanks

**Comment by jdk-21** [comment]
I open to adding Live TV in the future. But it isn't a high priority right now, because there are other issues that I want to focus on first. If somebody wants to contribute this functionality, I'm happy to merge it.
And please upvote this discussion. That way I can better estimate the demand for this feature.

**Reply by NetSven** [reply]
Ok, buti if you will do it, please put 1 important option what is missing as well from jellyfin app, but many people looking for that. Diffrent m3u playlist and epg list for diffrent users. That is very good idea, usually people get max 3 slots for 1 subscription, so if have 9 people need 3 subscription but at the moment I didn't see anywhere this future. A lot of people living in diffrent country and we have to use iptv for our languages channels.

**Reply by jdk-21** [reply]
Do you know if this possible with the Jellyfin API? I haven't worked with the TV part yet.
